logs/api/get.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# swift list production-juju-ps45-cdo-jujucharms-machine-1.canonical.com G 201610 G api.jujucharms.com.log > logs1.list
+# swift list production-juju-ps45-cdo-jujucharms-machine-2.canonical.com G 201610 G api.jujucharms.com.log > logs2.list
+
+source $HOME/archive-anon-cdo-jujucharms-ro.novarc
```

Comment:
Ah, I added that to the makefile but not to this script. Thanks. I'll update it.